import_export/management/commands/export.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+
+from import_export.command_utils import (
+    get_default_format_names,
+    get_format_class,
+    get_resource_class,
+)
+
+
+class Command(BaseCommand):
+    help = "Export data from a specified resource or model in a chosen format."
+
+    def add_arguments(self, parser):
+        default_format_names = get_default_format_names()
+        parser.add_argument(
+            "format",
+            help=f"""Specify the export format. Can be one of the default formats
```

Comment:
I'm wondering if all strings should support i18n ?

Reply:
I thought about this as well and concluded that 1) different type of users use management command  (as stated above) 2) other management commands do not support i18n (ie dumpdata)